plugins/mac-virtualcam/src/dal-plugin/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+# Universal build for Apple Silicon
+set(CMAKE_OSX_ARCHITECTURES "x86_64;arm64")
```

Comment:
According to https://cmake.org/cmake/help/v3.5/variable/CMAKE_OSX_ARCHITECTURES.html 

> The value of this variable should be set prior to the first project() or enable_language() command invocation because it may influence configuration of the toolchain and flags. It is intended to be set locally by the user creating a build tree.